online_store/gt_sport_v2.js

```diff
@@ -71,11 +71,16 @@ let productEl = [];
 const sportProductsEl = document.getElementById("sportProducts");
 const addToShoppingCartEl = document.getElementById("addToShoppingCart");
 let totalPriceEl = document.getElementById("totalPrice");
-let shoppingCartEl = document.getElementById("shoppingCart")
+let shoppingCartEl = document.getElementById("shoppingCart");
+let intoSCbtn = document.getElementById("intoSC");
+let calTPbtn = document.getElementById("calTP");
+let addToShoppingCartBtn = document.getElementById("addToShoppingCartBtn");
```

Comment:
Here it is getting the button element before it is being rendered.
Below you have the function **productDescriptionDisplay** which display the list of product with each button, so after that you have access to your button element.

Reply:
Sounds good